### ISSUE TITLE
Accounts are not showing in Mist0.8.5

### ISSUE THREAD

**Issue by prashantprabhakar** [ethereum/mist#1308]
#### System information
``````
Version: `0.8.5`
OS & Version: linux
Node type: `geth(default)`
``````
Accounts are not showing up in Mist 0.8.5.But I can see my accounts in keystore folder of .ethereum directory. I even added a new account to Mist, but that too did't show up but one more file was added in keystore representing newly created account.
Even on checking developer's tool of Mist, I can see all my accounts when I enter `web3.eth.accounts`.
So why accounts are not showing up in wallet? 
![no accounts](https://cloud.githubusercontent.com/assets/20419322/19425034/e5874b70-944b-11e6-8859-be6bc3c905af.png)

**Comment by luclu** [comment]
@prashantprabhakar could you please try to:
1. restart the wallet
2. add another account, click it and go back to the wallet screen

**Comment by prashantprabhakar** [comment]
Yes, I have already done both the steps. Whenever I create a new account, a new file is added in keystore. But nothing appears on Mist screen. I can see a generating wallet popup box after entering my password, and then nothing..A fresh wallet screen is in from of me without any accounts.

**Comment by optymusGrime** [comment]
I'm experiencing the same problem also followed steps outlined by @luclu. can verify the issue.

Windows x64 build (running on win 10)

**Comment by hiddentao** [comment]
Could you please open the Javascript Console of Mist (There is an menu entry to open the Developer tools) and paste here what you see.

**Comment by prashantprabhakar** [comment]
Console ouptut:

> Failed to load resource: net::ERR_FILE_NOT_FOUND file:///home/gemini/Ethereum/EthereumWallet0.8.5/Users/ich/github/mist/distâ€¦/wallet/packages/ethereum_dapp-styles/fonts/SourceSansPro-ExtraLightIt.otf Failed to load resource: net::ERR_FILE_NOT_FOUND

> Failed to load resource: net::ERR_FILE_NOT_FOUND  file:///sockjs/info?cb=nvxj0vet33 Failed to load resource: net::ERR_FILE_NOT_FOUND

> Node started syncing, stopping app operation @ file:///home/gemini/Ethereum/EthereumWallet0.8.5/Users/ich/github/mist/distâ€¦.asar/interface/wallet/f6e44fd3a6364a5106154cdf3516a4cbc1c24963.js:368:378

>

**Comment by hiddentao** [comment]
Hmm, not a lot of info to go on there. I’ll have to do some testing locally.

-- 
Ramesh Nair

Director
HiddenTao Ltd (UK company no. 6807289)

http://hiddentao.com
https://github.com/hiddentao
https://www.linkedin.com/in/hiddentao


On 17 October 2016 at 13:49:40, prashantprabhakar (notifications@github.com) wrote:

Console ouptut:

Failed to load resource: net::ERR_FILE_NOT_FOUND file:///home/gemini/Ethereum/EthereumWallet0.8.5/Users/ich/github/mist/distâ€¦/wallet/packages/ethereum_dapp-styles/fonts/SourceSansPro-ExtraLightIt.otf Failed to load resource: net::ERR_FILE_NOT_FOUND

Failed to load resource: net::ERR_FILE_NOT_FOUND file:///sockjs/info?cb=nvxj0vet33 Failed to load resource: net::ERR_FILE_NOT_FOUND

Node started syncing, stopping app operation @ file:///home/gemini/Ethereum/EthereumWallet0.8.5/Users/ich/github/mist/distâ€¦.asar/interface/wallet/f6e44fd3a6364a5106154cdf3516a4cbc1c24963.js:368:378

**Comment by ddsjangster** [comment]
I am experiencing the same issue with Ethereum Wallet v.0.8.6 as prasantprabhakar. i.e. Wallet is syncing, but the balance in my accounts shows 0 despite UTC files being in the correct keystore folder. What happened and how to fix it so I can see my correct ETH balance?

**Comment by jones-in-the-fast-lane** [comment]
This also happens to me (since 0.8.4 infact). Wallet file is in keystore.

win 7 64bit

**Comment by jones-in-the-fast-lane** [comment]
This also happens to me (since 0.8.4) and again in 0.8.6. Wallet file is in correct folder.

Win 7 64bit

**Comment by prashantprabhakar** [comment]
@hiddentao I am also  facing same issues with latest Ethereum Wallet 0.8.6. I think the issue lies in that Wallet cann't read files from keystore properly. Because after I removed everything and started downloading blockchain again, I created some accounts but none of them appeared on my wallet screen. but after restarting Mist, all are visible. Again I deleted 3 accounts out of 7 from keystore, but still Mist is showing all 7 accounts, If I check from geth I can see 4 accounts. SO geth and keystore are in agreement but Ethereum-Wallet isn't.

**Comment by nathan111100** [comment]
I'm having the same issue as well, my accoutns are all in the keystore but not showing up on mist
